pkg/bandwidth/bandwidth.go

```diff
@@ -63,6 +63,20 @@ func ProbeBandwidthManager() {
 		option.Config.EnableBandwidthManager = false
 		return
 	}
+	if option.Config.EnableBBR {
+		if h := probes.NewProbeManager().GetHelpers("sched_cls"); h != nil {
```

Comment:
Shouldn't we return a fatal error even for the `h == nil` case?

Reply:
Added a fatal if sched_cls helpers cannot be probed. Unlikely but makes sense to assert.